Pock/Widgets/ControlCenter/ControlCenterWidget.swift

```diff
@@ -138,8 +135,8 @@ extension ControlCenterWidget {
         guard let item = item else { return }
         slideableController = PKSlideableController.load()
         switch item.self {
-        case is CCVolumeUpItem, is CCVolumeDownItem:
-            slideableController?.set(downItem: volumeItems.first, upItem: volumeItems.last)
+        case is CCVolumeUpItem, is CCVolumeDownItem, is CCToggleMuteItem:
+            slideableController?.set(downItem: CCVolumeUpItem(parentWidget: self), upItem: CCVolumeDownItem(parentWidget: self))
```

Comment:
`downItem` should be `CCVolumeDownItem` and `upItem` should be `CCVolumeUpItem` 🌚

Reply:
fixed now🦄